src/guide/reusability/custom-directives.md

```diff
@@ -210,6 +210,11 @@ app.directive('demo', (el, binding) => {
 
 ## Usage on Components {#usage-on-components}
 
+:::warning Not recommended usage
```

Comment:
I think

Reply:
I agree with @AloisSeckar, `Not recommended` is enough 😅 

```suggestion
:::warning Not recommended
```